src/heatmap.ts

```diff
@@ -16,7 +16,7 @@ import {IVector} from 'phovea_core/src/vector';
 import {toSelectOperation, defaultSelectionType} from 'phovea_core/src/idtype';
 
 
-function defaultColor(value: any) {
+export function defaultColor(value: any) {
```

Comment:
I traced what value can be for a type. It should be defined in [`IDataDescription`](https://github.com/phovea/phovea_core/blob/master/src/datatype.ts#L17), but it isn't, because it depends on the data type. I checked the [index.json in TaCo](https://github.com/Caleydo/taco_server/blob/master/data/index.json#L9), which defines the `value` property for a matrix/`IMatrix`.

I suggest to define an optional `value` property in `IDataDescription` (similar to the `extra` property) with `type` and `range`.